assets/prometheus-k8s/prometheus.yaml

```diff
@@ -154,7 +157,7 @@ spec:
     image: quay.io/thanos/thanos:v0.10.0
     resources:
       requests:
-        cpu: 50m
+        cpu: 1m
```

Comment:
is this enough for thanos?

Reply:
This is for the thanos sidecar next to prometheus. In the e2e run I analyzed ~1millicore CPU:
![image](https://user-images.githubusercontent.com/375856/76525872-f18cbf00-646c-11ea-82c6-dba8e428a3db.png)